app/controllers/orders_controller.rb

```diff
@@ -27,6 +27,6 @@ def create
   private
 
   def order_params
-    params.permit(:name, :address, :city, :state, :zip)
+    params.permit(:name, :address, :city, :state, :zip).merge({user: current_user})
```

Comment:
I'm not familiar with this. I take it it's just merging the params of that current user?

Reply:
So to have an order save correctly, we need to pass along the current_user to associate the order with a user. Daniel and I were playing around with how to get this data into the Order.create call and learned that .merge works. There is probably a better way, though.